modules/web/.stylelintrc.yml

```diff
@@ -39,5 +39,4 @@ rules:
     - true
     - severity: warning
   scss/comment-no-empty:
-  string-quotes: single
```

Comment:
Just curious, why did we remove this?

Reply:
This rule is one of the [deprecated](https://stylelint.io/migration-guide/to-15/#:~:text=We%27ve%20deprecated%2076%20of%20the%20rules%20that%20enforce%20stylistic%20conventions%3A) rules in Stylelint v15

Reply:
now it's handled by our [Prettier](https://github.com/kubermatic/dashboard/blob/434357d073fe3645194a0e6bc5a246d621b1b77b/modules/web/.prettierrc.cjs#L28) configuration